Diffusion/Pipeline/PipelineLoader.swift

```diff
@@ -82,20 +90,20 @@ extension PipelineLoader {
         return url.lastPathComponent
     }
     
-    var downloadedPath: Path { PipelineLoader.models / filename }
-    var downloadedURL: URL { downloadedPath.url }
+    var downloadedURL: URL { PipelineLoader.models.appendingPathComponent(filename) }
 
-    var uncompressPath: Path { downloadedPath.parent }
+    var uncompressURL: URL { PipelineLoader.models.deletingLastPathComponent() }
+    
+    var packagesFilename: String { PipelineLoader.models.appendingPathComponent((filename as NSString).deletingPathExtension).path }
```

Comment:
```suggestion
    var packagesFilename: String { (filename as NSString).deletingPathExtension }
```

This should just be the `basename` (or filename), without the .zip extension.